js/search.js

```diff
@@ -174,6 +174,8 @@ search.addWidget(
         <div class="col-sm-3 col-lg-2">
             {{#meta}}
                 <p class="metadata">
+                    <span class="metadata-block"><i class="glyphicon glyphicon-tags"></i> {{ meta.latest_version }}</span>
+                    <span class="metadata-block"><i class="glyphicon glyphicon-calendar"></i> {{ meta.latest_date }}</span>
```

Comment:
what if those are not available ?

Reply:
i have not install/run the project locally for now, dont want to invest much time of PR is not accepted
so for now, it was just to make a decent screenshot with fake data ^^